starter-code/css/main.css

```diff
@@ -7,4 +7,98 @@ light blue: #1c5380
 body {
   font: 200 16px/1.5 "Helvetica Neue", Helvetica, Arial, sans-serif;
   margin: 0;
+  background-image : linear-gradient(to bottom, #1c5380 223px, #fff 203px);
 }
+
+.container{
+ 
+  display: flex;
+  justify-content: space-around;
+  width: 70%;
+  margin: 30px auto;
+}
+
+.img-left{
+  
+  padding: 20px;
+}
+
+.right{
+  
+  padding: 20px;
+}
+
+header{
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 50px;
+  margin-bottom: 15px;
+  padding: 0 9px;
+}
+
+.logo-insta{
+  height: 100%;
+  width: auto;
+}
+
+.image-appli{
+  display: flex;
+  justify-content: center;
+  margin-top: 25px;
```

Comment:
Usually you don't use `margin-top` too much. Margin is mostly used from to to bottom, meaning `margin-bottom`, or on the sides.
So if you need more space above your element, try giving it's upper neighbour some `margin-bottom` :)